### DISCUSSION TITLE
Is there a way to access logs from elsewhere than UI dashboard?

### DISCUSSION THREAD

**Discussion by jakubno** [supabase/supabase#20186] [Questions] [answered]
I want to setup alerting on error logs, but supabase AFAIK doesn't support this. 

So I wanted to create the alerting myself, but I can't get to the logs. I found old discussion on this topic: https://github.com/orgs/supabase/discussions/479, but `file_fdw` extension is no longer supported (maybe I am doing something incorrectly, but I can't create it with `postgres` user and the extension can't be turned on in UI dashboard)

I examined how the logs work in UI and it sends request to `api.supabase.io`, which needs `jwt` authentication and the token can be obtained only via login to the dashboard.

I hope there is a better way, because this doesn't sound like some exotic request.

Thanks in advance!

**Comment by lightstrike** [comment]
We're also interested in setting up an alerting system this way.

**Comment by fbatiga** [comment]
I created a ticket on this few months ago and the answer was no there is no way. It's on their roadmap, but without any indication of when it might land.

What i ended doing is wrap up all my edges functions with try catches and froward the logs to the database...

**Reply by lightstrike** [reply]
Thanks for the reply @fbatiga! We'll explore this.

**Comment by Ziinc** [comment, marked as answer]
We are currently working on a solution for this which will allow programmatic access in a sustainable fashion, please stay tuned. In the meantime, the only current way to access logs would be via the UI dashboard.

**Reply by ska-lbit** [reply]
> We are currently working on a solution for this which will allow programmatic access in a sustainable fashion, please stay tuned. In the meantime, the only current way to access logs would be via the UI dashboard.

Any update on this?

**Reply by wysohn** [reply]
Would love to hear an update on this

**Reply by lightstrike** [reply]
Likewise, this is still an area of interest for us.
